Tests/WeaverCodeGenTests/SwiftGenerator/Output/Weaver.test_nested_type.swift

```diff
@@ -0,0 +1,12 @@
+/// This file is generated by Weaver 0.12.4
+/// DO NOT EDIT!
+// MARK: - FooTest21
+protocol FooTest21DependencyResolver {
+    var foo: Test21Nest.Test21Type<.Test21Type> { get }
```

Comment:
Hello!
according to 
``` // weaver: foo <= Test21Nest.Test21Type ```
this line should be generated as
``` var foo: Test21Nest.Test21Type { get } ```

Reply:
Well spotted. I’ll take another look.

Reply:
You can also look through https://github.com/scribd/Weaver/pull/111 , I've faced such a problem too.

Reply:
Oh man, I’m sorry. I didn’t even see your pull request. I’ll close this one, yours is obviously more thorough. I’ll try to find time to review it just for a second pair of eyes.